powershell/plugins/llcsharp.ts

```diff
@@ -8,14 +8,25 @@ import { join } from 'path';
 import { Model } from '../llcsharp/code-model';
 import { State } from '../llcsharp/generator';
 import { Project } from '../llcsharp/project';
+import { Dictionary } from '@azure-tools/linq';
 
 const resources = `${__dirname}/../../resources`;
 
+function outerTest(service: Host, project: Project, fname: string, content: string) {
+  service.WriteFile(join(project.apifolder, fname), test(content, project.overrides), undefined, 'source-file-csharp');
+}
+
+function test(content: string, overrides: Dictionary<string>): string {
+  const a = applyOverrides(content, overrides);
+  return a;
+}
+
 export async function llcsharp(service: Host) {
   try {
     const project = await new Project(service).init();
 
-    await project.writeFiles(async (fname, content) => service.WriteFile(join(project.apifolder, fname), applyOverrides(content, project.overrides), undefined, 'source-file-csharp'));
+    // await project.writeFiles(async (fname, content) => service.WriteFile(join(project.apifolder, fname), test(content, project.overrides), undefined, 'source-file-csharp'));
```

Comment:
It is better we could align with other writeFiles function calls

Reply:
just for test. Add it back